platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/ProblemReportingCrossProjectModelAccess.kt

```diff
@@ -378,26 +364,6 @@ class ProblemReportingCrossProjectModelAccess(
             return super.getLogger()
         }
 
-        override fun getAnt(): AntBuilder {
```

Comment:
❓ Why did the Ant-related methods stop being a violation?

Reply:
It's declared as a violation in the parent, here it were redundant overrides